api/apps/api/src/modules/scenarios/locks/scenario.lock.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity(`scenario_locks`)
+export class ScenarioLockEntity {
+  @PrimaryColumn({
+    type: `uuid`,
+    name: `user_id`,
+  })
+  userId!: string;
+
+  @PrimaryColumn({
+    type: `uuid`,
+    name: `scenario_id`,
+  })
+  scenarioId!: string;
+
+  @Column({
+    type: `date`,
+    name: `grab_date`,
+  })
+  grabDate!: Date;
```

Comment:
[minor] just `createdAt` to keep convention

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/673/commits/4089893cc9412c2ea89e98dd68326c806486fa6d